### ISSUE TITLE
DVD ISO playback with direct path connection

### ISSUE THREAD

**Issue by cswrd** [Andy2244/jellyfin-androidtv-zidoo#4]
### Describe the feature you'd like

Hello, I am quite new to the entire jellyfin ecosystem at all. I like your fork. Thank you!
As far as I understood it uses zidoo's native player. With its home theatre app I can playback even DVD ISOs on my z9s. However, through this jellyfin app nothing happens. It'd be great if that would work at least when a direct path connection is present.

**Comment by Andy2244** [comment]
Yes its a known issue, technically it's possible to send the iso to the zidoo-player. Yet i think the Jellyfin by default handles iso's differently. Its on the bug-list and i know its a feature many want fixed.
